prover/src/lib.rs

```diff
@@ -538,11 +533,15 @@ impl<C: SP1ProverComponents> SP1Prover<C> {
         witness_stream.extend(input.write());
 
         runtime.witness_stream = witness_stream.into();
-        runtime.run();
```

Comment:
more canonical way to do this would be to do runtime.run().map_err(..)?;

Reply:
updated